index.html

```diff
@@ -56,6 +56,11 @@
                 return null;
             });
 
+            libController.on("sectionIconClicked", function (text) {
```

Comment:
Thanks @yeexinc for the update, please merge your PR with #73 and accordingly update the comments and use the constant field used to define the event name.